site/_redirects

```diff
@@ -0,0 +1 @@
+/swapi-graphql  https://graphql.github.io/swapi-graphql  200
```

Comment:
So this will do a https://graphql.github.io/swapi-graphql => https://graphql.org/swapi-graphql

Reply:
yes, as a 200 [netlify rewrite](https://www.netlify.com/docs/redirects/#rewrites-and-proxying), looks similar to an apache .htaccess rewrite rule or nginx `proxy_pass` rule